smart-contracts/index.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
```

Comment:
This is from the ZOS example.  Should test without this file or add a comment regarding it's purpose and source.

Reply:
When I added this it appeared to fix an issue but I believe that's a red herring and this file is not necessary.

Reply:
Yes, let's test without first? I believe this is only needed if using truffle exec.